src/renderers/server-render.jsx

```diff
@@ -184,10 +178,15 @@ const makeRenderer = (
 	const store = createStore(reducer, initialState);
 
 	// load initial config
-	store.dispatch(configureApiUrl(apiUrl));
-	store.dispatch(configureBaseUrl(host));
-	store.dispatch(configureLocaleCode(localeCode));
-
+	store.dispatch(
+		configure({
+			apiUrl,
+			baseUrl: host,
+			localeCode,
+			supportedLocaleCodes,
+			initialNow: new Date().getTime(),
```

Comment:
Turns out we don't have a unit test for this module, so I've opened a ticket to write that -
 we can make sure `initialNow` is one of the expected props of this function call there https://meetup.atlassian.net/browse/WP-341